src/connector/src/parser/postgres.rs

```diff
@@ -464,3 +520,112 @@ fn pg_numeric_to_string(val: Option<PgNumeric>) -> Option<String> {
         None
     }
 }
+
+#[derive(Clone, Debug)]
+struct EnumParser {
```

Comment:
What about naming this "EnumString"? Seems that other such `FromSql` impl types have normal data type names like `Uuid`, `PgNumeric`, etc.

Reply:
updated